packages/block-library/src/navigation/edit/index.js

```diff
@@ -303,6 +337,53 @@ function Navigation( {
 		// with the snapshot from the time when ref became undefined.
 	}, [ clientId, ref, innerBlocks ] );
 
+	useEffect( () => {
+		const setPermissionsNotice = () => {
+			if ( noticeRef.current ) {
+				return;
+			}
+
+			noticeRef.current = uniqueId( 'navBlockNoEditPermissions' );
+
+			createWarningNotice(
+				__(
+					'You do not have permission to edit this Navigation. Any edits made will not be saved.'
```

Comment:
'edit this Navigation' doesn't sound quite right to me, I think that's because in language 'navigation' isn't usually a tangible thing. I also don't think we use this term alone anywhere else.

I think we should qualify this by saying 'edit this Navigation block' or 'edit this Navigation menu', or possibly even just 'edit this block' or 'edit this menu'.

Reply:
I think the concept must be editing the Navigation Menu. I'll update that.